scripts/us_census/acs5yr/subject_tables/common/testdata/spec.json

```diff
@@ -0,0 +1,357 @@
+{
+
+	"populationType": {
+		"_DEFAULT": "Person",
+		"household": "Household",
+		"HOUSEHOLD": "Household"
+	},
+	"measurement": {
+		"Median age (years)": {
+			"populationType": "Person",
+			"measuredProperty": "age",
```

Comment:
Why does this have populationType? If we do specify it, then the "populationType" section seems redundant.

Reply:
I have removed the `populationTypes` from `measurement`